### DISCUSSION TITLE
Dynamic Links with custom domain stopped working

### DISCUSSION THREAD

**Discussion by Ggayane** [invertase/react-native-firebase#6608] [Q&A] [answered]
Dynamic Links which worked perfectly with customs domain for about half year, from yesterday they stoped to work. Whenever I open a link error screen appears with message: 

> Invalid Dynamic Link - Blocked
> We could not match param 'https://mythrillfiction.com?story=4unEt6O4tK2ctvtFxHB4' with whitelisted URL patterns in this Google project.
> If you are the developer of this app, ensure that your Dynamic Links domain is correctly configured and that the path component of this URL is valid.

Link example https://mythrill.app/share/GMXg?_imcp=1

I didn't change anything in the code, or firebase configs, can anyone tell what has been changed or what could cause this?

**Comment by mikehardy** [comment, marked as answer]
Did you try the link debugging facility? 

https://mythrill.app/share/GMXg?_imcp=1&d=1

(note the `&d=1` I added)

Did you read the error message carefully?

> could not match param '[https://mythrillfiction.com?story=4unEt6O4tK2ctvtFxHB4](https://mythrillfiction.com/?story=4unEt6O4tK2ctvtFxHB4)' with whitelisted URL patterns in this Google project.

Or what you see when you use the debugging facility

> We could not match param 'https://mythrillfiction.com?story=4unEt6O4tK2ctvtFxHB4' with whitelisted URL patterns in this Google project. [Learn more.](https://support.google.com/firebase/answer/9021429)

https://support.google.com/firebase/answer/9021429

Looks like they may have enabled link pattern allow lists by default :thinking: ? I suppose if you went and added a pattern that fits your links, it would work, based on the documentation available in your error message and the debugging facilities

**Reply by Ggayane** [reply]
Thanks for the answer! Yes they have changed the default, and adding all possible urls into whitelist solved the issue! (didn't even know about dynamic links whitelisted URL patterns until now though 🙂)

**Reply by mikehardy** [reply]
yeah - kind of a neat feature, changing the default is unexpected though, answering this yesterday caused me to go verify my domains and luckily I had set up the patterns when I originally made my links otherwise this would have snared me as well
